### PR TITLE
bug 1189432 - remove metro firefox

### PR DESCRIPTION
**Pull request by peterbe**
@rhelmer like this?

### CODE REVIEW

**Comment by rhelmer** [comment]
Pretty much. Need a migration too.

**Comment by peterbe** [comment]
Would it just be `DELETE from releases_recent where product_name='MetroFirefox'`?

**Comment by rhelmer** [comment]
@peterbe hmm well at minimum you would need to re-load the stored procedure you modified.

Here is an example:
https://github.com/mozilla/socorro/blob/master/alembic/versions/4a068f8838c6_bug_959731_update_product_versions_.py

**Comment by peterbe** [comment]
...as well as the delete? Delete first, then the `load_stored_proc(op, ['update_product_versions.sql'])` call?

**Comment by rhelmer** [comment]
I'm not sure what the purpose of the delete is. Do you want to let this age out?

AFAIK we've never tried to totally remove a product from Socorro before, I don't know what all needs to be removed. We let things like Camino age out.

File: alembic/versions/b99155654de_sunset_all_future_metrofirefox.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# revision identifiers, used by Alembic.
+revision = 'b99155654de'
+down_revision = '56f5cdf9bcdb'
+
+from alembic import op
+
+
+def upgrade():
+    op.execute("""
```
Comment (rhelmer) [inline alembic/versions/b99155654de_sunset_all_future_metrofirefox.py]: you also need `load_stored_proc(op, ['update_product_versions.sql'])`, to load the stored proc

File: alembic/versions/b99155654de_sunset_all_future_metrofirefox.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        UPDATE product_versions
+        SET sunset_date = NOW()
+        WHERE
+            product_name = 'MetroFirefox' AND
+            sunset_date >= NOW()
+    """)
+
+
+def downgrade():
```
Comment (rhelmer) [inline alembic/versions/b99155654de_sunset_all_future_metrofirefox.py]: I don't think it's worth trying to undo the product_versions update in the case of a downgrade, but you should load the stored proc again (the assumption being that if we downgrade, we want to revert the stored proc change), just `load_stored_proc(op, ['update_product_versions.sql'])` again

**Comment by peterbe** [comment]
`load_stored_proc(op, ['update_product_versions.sql'])` added

**Comment by rhelmer** [comment]
r+

We should try it out on stage, need to figure out exactly how to set up alembic so it works (we always used to run it on the DB as the postgres user, which isn't doable with RDS)

**Comment by peterbe** [comment]
Is alembic not automatically run? So now when I land this it won't execute the migration??

**Comment by rhelmer** [comment]
@peterbe alembic migrations have never been automatic, and we haven't had to run one in AWS yet. It should just be a matter of making an `alembic.ini` and running it somewhere that has access to the database.

**Pull request merged**